### PR TITLE
Refactor/tooltip  use v tooltip

### PR DESCRIPTION
**Pull request by RobinStut**

### CODE REVIEW

File: src/heads-up/components/app-tooltip.vue [hunk 1]
```diff
@@ -15,234 +15,214 @@
...
-    border: 1px solid var(--base-color);
-  }
+.tooltip[aria-hidden="false"] {
+  visibility: visible;
+  opacity: 1;
+  transition: opacity 0.15s;
+  box-shadow: 0px 0px 10px -4px rgba(0, 0, 0, 1),
+    0px 0px 6px -4px rgba(0, 0, 0, 1);
+}
+/*
```
Comment (petergoes) [inline src/heads-up/components/app-tooltip.vue]: All commented out code can be removed, if it is needed back again, we can look it up in this commit

File: src/heads-up/components/app-tooltip.vue [hunk 2]
```diff
@@ -1,152 +1,101 @@
 <template>
-  <span
-    v-tooltip.top="{ ref: 'tooltip' }"
+  <v-popover
+    trigger="hover"
+    :placement="placement"
+    :delay="{ show: 500, hide: 100 }"
```
Comment (petergoes) [inline src/heads-up/components/app-tooltip.vue]: The `hide` in my commits were merely to test the functionality, I think you can leave it out.

File: src/heads-up/components/properties-item.vue [hunk 3]
```diff
@@ -5,7 +5,10 @@
         <slot />
       </span>
 
-      <app-tooltip class="properties-item__tooltip">
+      <app-tooltip
+        class="properties-item__tooltip"
+        :placement="'top-start'"
```
Comment (petergoes) [inline src/heads-up/components/properties-item.vue]: Please use a static string here, so `placement="top-start"`. This way, it is overly verbose

File: src/heads-up/components/properties-item.vue [hunk 4]
```diff
@@ -35,13 +38,17 @@
...
+        :class="{
+          'properties-item__strike': errors && !valueWithExceededLength
+        }"
       >{{ value }}</span>
       <span v-if="valueWithExceededLength">{{ valueMinusExceededLength }}</span>
       <span
         v-if="valueWithExceededLength"
         class="properties-item__strike"
-      >{{ valueWithExceededLength }}</span>
+      >{{
```
Comment (petergoes) [inline src/heads-up/components/properties-item.vue]: I found the previous version (on one line better readable)

File: src/heads-up/components/app-tooltip.vue [hunk 5]
```diff
@@ -1,152 +1,101 @@
...
-    .vue-tooltip {
-      width: 300px;
-    }
-  }
+.tooltip[aria-hidden="false"] {
+  visibility: visible;
+  opacity: 1;
+  transition: opacity 0.15s;
+  box-shadow: 0px 0px 6px 1px rgba(0, 0, 0, 0.2);
+  width: 150px;
```
Comment (petergoes) [inline src/heads-up/components/app-tooltip.vue]: Where does this value come from? I find the tooltip very narrow. instead I'd perfer a `max-width` of `75vw`.

**Review by petergoes — CHANGES_REQUESTED** [review]
In addition to the things below, I miss a dark theme tooltip

File: src/heads-up/components/properties-item.vue (hunk 4 above)
Comment (RobinStut) [reply src/heads-up/components/properties-item.vue]: This was formatted on save in my code editor, had to disable it to fix it

File: src/heads-up/components/properties-item.vue [hunk 6]
```diff
@@ -171,74 +156,58 @@ export default {
...
-
 .properties-item__link {
   margin-left: 5px;
 }
-
 @media (min-width: 500px) {
   .properties-item {
     display: flex;
-    align-items: flex-start;
+    align-items: center;
```
Comment (petergoes) [inline src/heads-up/components/properties-item.vue]: here

File: src/heads-up/components/properties-item.vue [hunk 7]
```diff
@@ -171,74 +156,58 @@ export default {
...
     line-height: 1.4em;
   }
-
   .properties-item__term {
-    display: flex;
-    justify-content: flex-end;
+    display: unset;
     width: var(--term-width-small);
     padding-right: 5px;
+    text-align: right;
```
Comment (petergoes) [inline src/heads-up/components/properties-item.vue]: here

File: src/heads-up/components/properties-item.vue [hunk 8]
```diff
@@ -171,74 +156,58 @@ export default {
...
   }
-
-  .properties-item__term * + * {
-    margin-left: 0.15rem;
-  }
-
   .properties-item__description {
```
Comment (petergoes) [inline src/heads-up/components/properties-item.vue]: here

**Review by petergoes — CHANGES_REQUESTED** [review]

**Review by petergoes — APPROVED** [review]

**Pull request merged**